src/js/components/Drop/stories/Plain.js

```diff
@@ -1,36 +1,36 @@
 import React, { useEffect, useRef, useState } from 'react';
 
-import { Box, Drop, Grommet } from 'grommet';
-import { grommet } from 'grommet/themes';
+import { Box, Drop } from 'grommet';
 
 const PlainDrop = () => {
   const targetRef = useRef();
 
   const [, setShowDrop] = useState(false);
   useEffect(() => setShowDrop(true), []);
   return (
-    <Grommet theme={grommet} full>
```

Comment:
(starting to get sad about this haha)
@jcfilben just pointing out another case of `full`, as you mentioned it on another PR.

Reply:
This story we will want to add the `full` prop on. We will have to wait for #5738.